generator/transit_generator.hpp

```diff
@@ -1,9 +1,94 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "base/macros.hpp"
+
+#include "3party/jansson/myjansson.hpp"
+
+#include <cstdint>
 #include <string>
+#include <type_traits>
+#include <vector>
 
 namespace routing
 {
 namespace transit
 {
-void BuildTransit(std::string const & mwmPath, std::string const & srtmDir);
+class DeserializerFromJson
```

Comment:
What prevents us from moving this class to the .cpp file?

Reply:
Only tests: omim/generator/generator_tests/transit_test.cpp